content/publishing-yaml/distribution.md

```diff
@@ -86,7 +86,7 @@ publishing:
 ```
 
 {{<notebox>}}
-The proper way to add your keys in `codemagic.yaml` is to copy the contents of the key file and [encrypt](../building/encrypting) it. Then add the encrypted value into the configuration file.
+You can override the publishing track specified in the configuration file using the environment variable `GOOGLE_PLAY_TRACK`. This is useful if you're starting your builds via [Codemagic API](../rest-api/overview/) and want to build different configurations without editing the configuration file.
 {{</notebox>}}
```

Comment:
The variable should also be documented in https://docs.codemagic.io/building/environment-variables/

Reply:
it's only for read-only env vars, ones that codemagic specifies itself and user can use

Reply:
OK, in this case we should just mention that the accepted values of the env variable are the same as in yaml config (I assume?)

Reply:
that's what i meant, track is taken from yaml config, but it can be overriden with the env var